Docs for the 0.22 release (#2307)
kestra-io/docs: content/docs/expressions/index.md

To use an environment variable, such as `KESTRA_FOO`, reference it as `{{ envs.foo }}`. The variable name is derived by removing the `KESTRA_` prefix and converting the remainder to **lowercase**.

> To access [Environment configuration variables](../configuration/index.md#environment), you can use `{{ kestra.environment.name }}` to reference the name given to your environment or `{{ kestra.url }}` to reference the environment's configured URL.

Reviewer: ```suggestion
To reference the [environment name](../configuration/index.md#environment) defined in Kestra configuration, you can use `{{kestra.environment.name}}`. Similarly, using `{{kestra.url}}`, you can reference the environment's URL set in your Kestra configuration.
```

```yaml
tasks:
  - id: port
    type: io.kestra.plugin.core.log.Log
    message: "Generated Port: {{ randomPort() }}"
```

### fileSize

> The `fileSize` function returns the size of the file present at the given uri location.

Reviewer: ```suggestion
The `fileSize` function returns the size of the file present at the given URI.
```

Example:

```yaml
tasks:
  - id: download
    type: io.kestra.plugin.core.http.Download
    uri: https://huggingface.co/datasets/kestra/datasets/raw/main/csv/orders.csv
  
  - id: fileSize
    type: io.kestra.plugin.core.log.Log
    message: "The file size is {{ fileSize(output.download.uri) }}"
```

### fileExists

> The `fileExists` function returns true if the file is present at the given uri location.

Reviewer: ```suggestion
The `fileExists` function returns `true` if the file is present at the given URI.
```

```yaml
tasks:
  - id: download
    type: io.kestra.plugin.core.http.Download
    uri: https://huggingface.co/datasets/kestra/datasets/raw/main/csv/orders.csv

  - id: fileExists
    type: io.kestra.plugin.core.log.Log
    message: "The file exists: {{ fileExists(output.download.uri) }}"
```

### fileEmpty

> The `fileEmpty` function returns true if the file present at the given uri location is empty.

Reviewer: ```suggestion
The `fileEmpty()` function returns true if the file present at the given URI is empty.
```

```yaml
tasks:
  - id: download
    type: io.kestra.plugin.core.http.Download
    uri: https://huggingface.co/datasets/kestra/datasets/raw/main/csv/orders.csv

  - id: fileEmpty
    type: io.kestra.plugin.core.log.Log

>     message: "Is the file empty: {{ fileEmpty(output.download.uri) }}"

Reviewer: ```suggestion
    message: "Is the file empty? {{ fileEmpty(output.download.uri) }}"
```
